src/mame/drivers/gigatron.cpp

```diff
@@ -75,7 +81,7 @@ void gigatron_state::gigatron(machine_config &config)
 
 	/* video hardware */
 	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
-	screen.set_refresh_hz(60);
+	screen.set_refresh_hz(59.98);
```

Comment:
Why are you not using raw screen parameters?  You have the schematics for this machine, don’t you?  You should be able to derive the screen parameters from the clocks.

Reply:
I haven't examined the schematics for this particular machine yet (I believe it involves some sort of VGA chipset), but even I can't always figure out screen parameters from schematics. One of the most intractable cases I've come across is PolyMorphic Systems' VTI, whose dot clock is generated by a VCO connected to a potentiometer and has no bearing on the sync rates, which are divisions of either an optional onboard XTAL or the bus clock... and the Poly-88 has a notably nonstandard value for the latter.

Reply:
The Gigatron's video signal i generated by software. That is the whole point of the machine....